kaazing-gateway/README.md

```diff
@@ -18,20 +18,22 @@ The Kaazing Gateway is a network gateway created to provide a single access poin
 
 By default the gateway runs a WebSocket echo service similar to [websocket.org](https://www.websocket.org/echo.html).
 
+You must give your gateway container a hostname. To do this, use the *docker run* -e option to define an environment variable, GATEWAY_OPTS, along with the -h option (your hostname may vary):
+
 ```console
-$ docker run --name some-kaazing-gateway -h somehostname -d -p 8000:8000 kaazing-gateway
+$ docker run --name some-kaazing-gateway -e GATEWAY_OPTS="-Dgateway.hostname=somehostname" -h somehostname -d -p 8000:8000 kaazing-gateway
```

Comment:
To preserve the -Djava.security.egd=file:/dev/urandom needed by docker perhaps we should add that to the GATEWAY_OPTS line.